client/src/game/ui/notes.ts

```diff
@@ -170,9 +170,9 @@ export function openEditTooltip(card: HanabiCard): void {
         );
       }
 
-      // Strip any HTML elements
+      // Convert &, <, >, " to HTML entities
       // (to be thorough, the server will also perform this validation)
```

Comment:
This comment is now out of date right?
```suggestion
      // Convert &, <, >, " to HTML entities
      // (to be thorough, the server will also validate that it contains no HTML tags)
```